types/src/auction/bid/mod.rs

```diff
@@ -196,6 +207,18 @@ impl Bid {
         vesting_schedule.initialize(staked_amount)
     }
 
+    /// Sets given bid's `inactive` field to `false`
+    pub fn activate(&mut self) -> bool {
+        self.inactive = false;
```

Comment:
Personal preference nitpick: Unless I'm missing something special about this name I'd prefer "active" name over "inactive" as with "inactive" the meaning is already negated:

```rust
pub fn activate(&mut self) {
        self.active = true;
}
```

And also in logic above doesn't need negation

```diff
- .filter(|(_public_key, bid)| bid.vesting_schedule().is_some() && !bid.inactive())
+ .filter(|(_public_key, bid)| bid.vesting_schedule().is_some() && bid.is_active())
```

Reply:
I chose `inactive` because the default value, `false`, should be the default case: all bids are active by default.

Reply:
I prefer it the way henry has it, as the starting and most common state is `active`; thus defaulting to inactive: false (default value of bool) makes the most logical sense to me.

Reply:
ninja'd